pkg/packagekit/updates.jsx

```diff
@@ -449,57 +555,259 @@ class ApplyUpdates extends React.Component {
+                    <TextContent>
+                        <TextList>
+                            {col1.map(service => (<TextListItem key={service}>{service}</TextListItem>))}
+                        </TextList>
+                    </TextContent>
+                </FlexItem>
+                {col2.length > 0 && <FlexItem flex={{ default: 'flex_1' }}>
+                    <TextContent>
+                        <TextList>
+                            {col2.map(service => (<TextListItem key={service}>{service}</TextListItem>))}
+                        </TextList>
+                    </TextContent>
+                </FlexItem>}
+            </Flex>
+        );
+
+        const title = (<>
+            <ProcessAutomationIcon />
+            <span className="update-success-table-title">
+                {_("Some sofware needs to be restarted manually")}
+            </span>
+        </>);
+
+        entries.push({
+            columns: [
+                { title },
+            ],
+            props: { key: "manual" },
+            hasPadding: true,
+            expandedContent,
+        });
+    }
+
+    return (<>
+        <EmptyStatePanel title={ _("Update was successful") }
+            secondary={
+                <>
+                    { entries.length > 0 && <ListingTable aria-label={_("Update Success Table")}
+                        columns={[{ title: _("Info") }]}
+                        showHeader={false}
+                        className="updates-success-table"
+                        rows={entries} /> }
+                    <div className="update-success-actions">
+                        { reboot.length > 0 && <Button id="reboot-system" variant="primary" onClick={openRebootDialog}>{_("Reboot system...")}</Button> }
+                        { restart.length > 0 && <Button id="choose-service" variant={reboot.length > 0 ? "secondary" : "primary"} onClick={openServiceRestartDialog}>{_("Restart services...")}</Button> }
+                        { reboot.length > 0 || restart.length > 0
+                            ? <Button id="ignore" variant="link" onClick={onIgnore}>{_("Ignore")}</Button>
+                            : <Button id="ignore" variant="primary" onClick={onIgnore}>{_("Continue")}</Button> }
+                    </div>
+                </>
+            } />
+        <div className="flow-list-blank-slate">
+            <Expander title={_("Package information")}>
+                <PackageList packages={history[0]} />
+            </Expander>
+        </div>
+    </>);
+};
 
+const StatusCard = ({ updates, highestSeverity, timeSinceRefresh, restartableServices, onValueChanged }) => {
+    const numUpdates = Object.keys(updates).length;
+    const numSecurity = count_security_updates(updates);
+    const numRestartServices = restartableServices.daemons.length;
+    const numManualSoftware = restartableServices.manual.length;
+    const numRebootServices = restartableServices.reboot.length;
```

Comment:
Would "RebootPackages" or "RebootApplications" fit better? "reboot services" sounds really confusing, as a service can be restarted.